platforms/dotnet/versions/buster/defaultVersion.txt

```diff
@@ -0,0 +1 @@
+3.1
```

Comment:
nit: just curious, are the default versions across Debian flavors for a given platform going to start off the same? I'm wondering if there's a case that we need to handle where the default for bullseye is different from buster, for example.

Reply:
@cormacpayne  So this file was added for each OS type exactly for that case. I was imagining a scenario in the future where we have stopped releasing versions to stretch as it is EOL, and we want to update the bullseye image's default version for a language BUT that new version doesn't exist for stretch.

This file will get uploaded to the storage account as `defaultVersion.buster.txt`, and our CLI can pull it down if its running on buster.